### DISCUSSION TITLE
add_netcdf throws decode time error

### DISCUSSION THREAD

**Discussion by mn5hk** [opengeos/leafmap#272] [Q&A] [answered]
Hi,

I am trying to visualize an xarray file with time units defined as 'months since 1901-01-01' with "calendar 'proleptic_gregorian'.

I can open the netcdf file using leafmap.read_netcdf (or xarray) as: 
```

leafmap.read_netcdf(netcdf_filename, decode_times = False)

```

However, loading the netcdf file onto map as:
```

m.add_netcdf(netcdf_filename, decode_times = False)

```

throws the following error:

ValueError: unable to decode time units 'months since 1901-01-01' with "calendar 'proleptic_gregorian'". Try opening your dataset with decode_times=False or installing cftime if it is not installed.

I have tried to install cftime, however it does not help. Is there any workaround?

Link to sample netcdf file: https://hs.pangaea.de/model/WaterGAP_v2-2d/watergap_22d_WFDEI-GPCC_histsoc_tws_monthly_1901_2016.nc4

**Comment by giswqs** [comment, marked as answer]
Under the hood, the netcdf is converted to geotiff in order to be displayed on the map. You can convert it to GeoTIFF manually then use `m.add_geotiff()` to display it on the map.
